src/user_guide/singularity.ipynb

```diff
@@ -736,6 +736,51 @@
     "### Other `singularity exec` options"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "source": [
+    "You can use parameter `entrypoint` to execute `interpreter`, `entrypoint` used to set up the primary command for the container or to define a default shell or interpreter to use for running scripts\n",
+    "\n",
+    "```SoS\n",
+    "entrypoint = 'micromamba run -n ENVIRONMENT'\n",
+    "```\n",
+    "It activates a certain environment. For example, the following statement will execute `interpreter` Rscript"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "kernel": "SoS"
+   },
+   "outputs": [],
+   "source": [
+    "R: entrypoint = 'micromamba run -n ENVIRONMENT'"
```

Comment:
What are the output of the commands? The documentation is supposed to show the output of most of the commands.

Reply:
Thank you for your comments. My understanding is that `R: entrypoint = 'micromamba run -n ENVIRONMENT'`  is a statement not a command

Reply:
If `micromamba` works in your local environment,

```
R: entrypoint = 'micromanda run -n ENVIRONMENT'
a-simple-R-script
```
will execute the R script within the specified environment. We just need to show users how to use this parameter.